tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/event-map', 'Integration | Component | public/event map', {
```

Comment:
Please update this component test file according to the other component test files.